docs/using-borges/key-concepts.md

```diff
@@ -10,7 +10,7 @@ A standalone process that takes URLs from RabbitMQ, clones remote repository and
 
 ## Borges packer
 
-A standalone process that takes repository paths (or URLs) from a file and packs them into siva files (as a *Rooted Repository*) in the given output directory.
+A standalone process that takes repository paths (or URLs) from a file and packs them into siva files (as a *Rooted Repository*) in the given output directory. The producer will automatically do this when downloading repositories.
```

Comment:
separate section about producer?

> The producer will automatically do this...
Hmm this one should sound somehow better

Reply:
Do you mean adding that comment in the producer section instead of in the pack one?

Reply:
Maybe explaining that this command is a combined producer and consumer?

> This command does the same job as a files producer and consumer together without queue or database dependency.